src/server/routes/users.js

```diff
@@ -135,7 +137,7 @@ router.post('/create', adminAuthMiddleware('create a user.'), async (req, res) =
 });
 
 /**
- * Route for updating user role
+ * Route for updating an existing user.
  */
 router.post('/edit', adminAuthMiddleware('update a user role'), async (req, res) => {
```

Comment:
I realize you did not do this but could you change description to 'edit a user'.